docs/source/roadmap.md

```diff
@@ -0,0 +1,266 @@
+
+### Local processing of data for SpaceTx Users
+Some SpaceTx Assays have moderate processing requirements. Users of these assays would like to be able to use _starfish_ to process multiple fields of view of SpaceTx data on their local machines to enable further parameter tuning and spot-checking of results. To support this, we will define a Python API for running _starfish_ on local machines that processes experiments of arbitrary numbers of fields of view, logs, stores data provenance, is multi-processing enabled. Users will also be able to run _starfish_ pipelines through a command-line interface (CLI).
+
+### Scalable processing of SpaceTx assays
+For the SpaceTx consortium project, the _starfish_ team agreed to process the data for each of the data contributors. It may not always be possible or time efficient to do so on a local machine. To accomplish this, _starfish_ must be able to process a complete experiment consisting of hundreds of fields of view that are comprised of up to 10s of terabytes of 2-d TIFF images ("Scalable Pipeline Runner"). We will run these analyses, so can dictate the infrastructure on which this processing will occur. However, we need to expose tooling to explore the output formats on a personal computer of our contributor's choosing. The Chan Zuckerberg Biohub workflow is maintained by a single local user, for whom we can help set up this workflow runner.
+
+### Scientific assessment of analyzed SpaceTx data: cell type mapping
+The SpaceTx consortium Working Group 5 is devoted to mapping the cells in Cell x Gene tables into the transcriptomic types/clusters delivered by Working Group 2.  _starfish_ will define standard output formats for external (non-_starfish_) calculation of these cell types and subsequent incorporation of cell type information as metadata in the Cell x Gene tables.
+
+### Scientific assessment of analyzed SpaceTx data: methods comparison and meta analysis
+The SpaceTx consortium Working Group 6 is devoted to comparing the results of each method and assessing their relative strengths and weaknesses. These are largely scientific analyses enabled by the standardization of each assay's outputs by _starfish_.  Because each assay outputs an IntensityTable, Segmentation Mask, and Expression Matrix, it will be simple to label cells or spots with anatomical information represented as named 2D polygons.
+
+This capability will allow straightforward creation of highly compressed datasets that will enable working Group 6 to ask questions across all methods like "how many cells/mm2 belonging to the Sst-Chodl cluster were found in layer 4 of mouse primary visual cortex?"  or "Within the inhibitory cell class, what is the abundance and distribution of Pvalb+ and VIP+ cells in human cortex?"
```

Comment:
Is this clearer? 
```
Starfish's capability to create these highly compressed output formats will enable 
working Group 6 to ask questions across all methods like... 
```

Reply:
I think this helps, but I think it's still unclear what is meant by "highly compressed"? Is the idea that the output data are much smaller because the images have been transformed into a set of features (i.e., decoded spots, cell outlines, and expression matrices)? Or is the main point that the standardized output allows direct comparison across experiments, whereas when each group uses their own analysis package, the data needs to be wrangled first?

Reply:
It's both, good point. I've amended this section.